roles/splunk_common/tasks/enable_forwarder_monitoring.yml

```diff
@@ -0,0 +1,27 @@
+- name: Enable forwarder monitoring
+  ini_file:
+    path: "{{ splunk.home }}/etc/apps/splunk_monitoring_console/local/savedsearches.conf"
+    section: "DMC Forwarder - Build Asset Table"
+    option: "{{ item.key }}"
+    value: "{{ item.value }}"
+  with_dict: {"cron_schedule": "{{ dmc_asset_interval }}", "alert.track": 0, "disabled": 0,
```

Comment:
Looking at this again, I don't think we need to update the visualizations section here, so we can probably take it out.  Shutting off the run_on_startup we should probably also set through the REST endpoint, and then we can likely avoid triggering the restart handler.

Reply:
I looked back at the default savedsearches.conf file and you're correct, the visualizations and run_on_startup settings are already set to these values.
The one thing I'm unsure about is the restart handler because splunk was having trouble finding the search unless I triggered a restart. Keeping run_on_startup disabled also keeps the behavior consistent with this process through the UI.

Reply:
Okay, sounds good.  Lets undo the visualization update but we can keep the restart trigger

Reply:
I wasn't able to get it working without the restart trigger, but I removed the settings from the request body that are already assigned correctly.